libraries/AC_DroneShowManager/AC_DroneShowManager.cpp

```diff
@@ -353,6 +353,7 @@ const AP_Param::GroupInfo AC_DroneShowManager::var_info[] = {
 static DroneShowLEDFactory _rgb_led_factory_singleton;
 
 static bool is_safe_to_change_start_time_in_stage(DroneShowModeStage stage);
+static bool are_almost_equal(float a, float b);
```

Comment:
Compiler didn't like comparisons of variables to float values. Maybe there is a utility that already provides this?